src/test/scala/uk/gov/hmcts/ccd/corecasedata/scenarios/GetPaginationMetaData.scala

```diff
@@ -12,14 +13,17 @@ object GetPaginationMetaData extends PerformanceTestsConfig {
   def httpRequest() = {
     val s2sToken = CcdTokenGenerator.generateGatewayS2SToken()
     val userToken = CcdTokenGenerator.generateWebUserToken(url)
-    http("get pagination metadata data")
+    http("TX06_CCD_GetPaginationMetadataData")
       .get(url)
       .header("ServiceAuthorization", s2sToken)
       .header("Authorization", userToken)
       .header("Content-Type","application/json")
       .check(status in  (200))
   }
 
-  val getPaginationMetaData = scenario("Get Pagination Metadata")
-    .exec(httpRequest())
+  println("GetCaseData: Minimum think time " + MinThinkTime + " Maximum think time " + MaxThinkTime)
```

Comment:
can we use String interpolation for long strings please:
```
println(s"GetCaseData: Minimum think time $MinThinkTime Maximum think time $MaxThinkTime")
```

Reply:
OK, I am adding this comment in my to-do list and will do that in next PR.